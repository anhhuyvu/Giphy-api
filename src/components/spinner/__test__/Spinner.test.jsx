import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';

import Spinner from '../Spinner';

describe('Spinner Component', () => {
  it('should render snapshot correctly', () => {
    const tree = renderer.create(<Spinner />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('should exist spinner', () => {
    const { container } = render(<Spinner />);

    expect(container.getElementsByClassName('lds-roller')).toBeDefined();
  });
});
