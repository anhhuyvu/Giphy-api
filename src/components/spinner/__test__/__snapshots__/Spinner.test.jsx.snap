// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component should render snapshot correctly 1`] = `
<div
  className="lds-roller"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
